pkg/config/config.go

```diff
@@ -98,18 +98,21 @@ func validateConfig(config Config) error {
 		return fmt.Errorf("you must set heartbeat.interval")
 	}
 
-	if config.Conference.HeartbeatConfig.Timeout > 30 {
-		return fmt.Errorf("heartbeat.timeout must be 30s or lower")
+	// MSC3898 dictates what are the limits for these values.
+	if config.Conference.HeartbeatConfig.Timeout < 30 {
+		return fmt.Errorf("heartbeat.timeout must be 30s or higher")
```

Comment:
Do we really want it to be lower than 30 seconds? - If we do so, it means that the overall timeout when sending pings would be `interval + timeout` because after sending each ping we don't send a new ping before receiving a pong.

Or is it expected that we don't wait for confirmations for each pong and just shoot pings using the interval disregarding timeout? - If so, we would need to change our implementation (currently it's ping that expects an acknowledgment, i.e. pong). Another alternative is to make each side send **a unidirectional heartbeat** meaning that `ping`s are heartbeats that the server sends to the client so that the client can detect when the server is dead. While `pong`s are "client issues pings" essentially, the server expects to come each `timeout` (then we would need to rename values also).

Reply:
From the MSC it's not 100% clear to me... @dbkr, what are your thoughts on this?

Reply:
Sorry for not making this super clear - it's the client side timeout that needs to be greater than 30s (because that's how often the server will be sending ping packets). The server side timeout just needs to be long enough that the server is sure the client isn't there any more - I'd suggest maybe 10 seconds?

I'd expect us to fire pings regularly and not wait for a pong response before starting the timer - that way the maximum 30s interval is maintained.

Would some suggested values in the MSC help make this clearer?